plugins/hosts/linux/plugin.rb

```diff
@@ -26,13 +26,6 @@ class Plugin < Vagrant.plugin("2")
         Cap::NFS
       end
 
-      # Linux-specific helpers we need to determine paths that can
-      # be overriden.
-      host_capability("linux", "nfs_apply_command") do
-        require_relative "cap/nfs"
-        Cap::NFS
-      end
```

Comment:
Because NFS export and apply are no longer separate actions. Everything is now done `nfs_export` capability.